apps/api/src/app/services/search/search.service.ts

```diff
@@ -12,19 +12,28 @@ import { ContentKind, ContentModel } from '@dragonfish/shared/models/content';
 
 @Injectable()
 export class SearchService implements ISearch {
+    INITIAL_PAGE = 1;
+    INITIAL_MAX_PER_PAGE = 6;
+    MAX_PER_PAGE = 15;
+
```

Comment:
As these numbers are repeated often throughout this class, I made them fields.

Reply:
always a fan. 👍 
nitpick: make 'em const too, to clarify intent